models/provider.go

```diff
@@ -6,6 +6,15 @@ type DNSProvider interface {
 	GetDomainCorrections(dc *DomainConfig) ([]*Correction, error)
 }
 
+// DNSProvider will replace DNSProvider in 3.0.
```

Comment:
Remove comment again.

Reply:
I've updated it to explain what's going on to future provider authors. PTAL